flashinfer/gemm/gemm_mm_mxfp8_cute_dsl.py

```
@@ -0,0 +1,291 @@
+            )
+
+            make_kernel = lambda: Sm120B12xBlockScaledDenseGemmKernel(
+                sf_vec_size,
+                mma_tiler_mn,
+                cluster_shape_mn,
+                mma_k=32,
+                tile_k=128,
+                use_prefetch=use_prefetch,
+                enable_pdl=enable_pdl,
+                swap_ab=swap_ab,
+            )
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `make_kernel` as a `def`

(E731)